README.md

```diff
@@ -1,48 +1,17 @@
 # Juno.jl
 
+[![Developer Chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/JunoLab/Juno)
 [![Build Status](https://travis-ci.org/JunoLab/Juno.jl.svg?branch=master)](https://travis-ci.org/JunoLab/Juno.jl)
 
-This package defines [Juno](http://junolab.org/)'s frontend API (to install Juno, follow the instructions [here](https://github.com/JunoLab/uber-juno/blob/master/setup.md)). It is aimed primarily at allowing package authors to:
+This package defines [Juno](http://junolab.org/)'s frontend API and is the user facing repository for the project.
+Please report your issues either at the [Juno Discussion Board](http://discuss.junolab.org/)
```

Comment:
I'd move this down to just below the intro, otherwise it breaks the flow a little.